spark/src/test/scala/org/apache/spark/sql/delta/DeltaVacuumSuite.scala

```diff
@@ -66,6 +66,15 @@ trait DeltaVacuumSuiteBase extends QueryTest
     }
   }
 
+  protected def withEnvironment(prefix: String)(f: (File, ManualClock) => Unit): Unit = {
+    withTempDir(prefix) { file =>
+      val clock = new ManualClock()
+      withSQLConf("spark.databricks.delta.retentionDurationCheck.enabled" -> "false") {
+        f(file, clock)
+      }
```

Comment:
can you make a private helper shared by this and the other `withEnvironment`? Also, let's replace the string with `DeltaSQLConf.DELTA_VACUUM_RETENTION_CHECK_ENABLED.key`